rules/integrations/jamf/credential_access_high_volume_of_pbpaste.toml

```diff
@@ -0,0 +1,129 @@
+[metadata]
+creation_date = "2024/08/27"
+integration = ["jamf_protect"]
+maturity = "production"
+updated_date = "2024/08/27"
+
+[transform]
+[[transform.investigate]]
+label = "Show events having the same reponsible process"
+providers = [
+  [
+    {field = "host.hostname", excluded = false, queryType = "phrase", value = "{{host.hostname}}", valueType = "string"},
+    {field = "process.entity_id", excluded = false, queryType = "phrase", value = "{{process.group_leader.entity_id}}", valueType = "string"}
+  ]
+]
+
+[[transform.investigate]]
+label = "Show events having the same parent process"
+providers = [
+  [
+    {field = "host.hostname", excluded = false, queryType = "phrase", value = "{{host.hostname}}", valueType = "string"},
+    {field = "process.entity_id", excluded = false, queryType = "phrase", value = "{{process.parent.entity_id}}", valueType = "string"}
+  ]
+]
+
+[rule]
+author = ["Thijs Xhaflaire"]
+description = """
+Identifies a high volume of `pbpaste` executions, which may indicate a bash loop continuously collecting clipboard contents, potentially allowing an attacker to harvest user credentials or other sensitive information.
+"""
+from = "now-9m"
+index = ["logs-.*"]
```

Comment:
Can you update this to be based on the integration? Should be something like `logs-jamf_protect*`